### ISSUE TITLE
The implementation correction

### ISSUE THREAD

**Issue by avishai111** [tomastokar/Additive-Margin-Softmax#1]
Hi, excellent article implementation.
But I think you have a bug.
The logits in AMSLOSS should be cos_theta. The Logists you return are m_Logists that go into the loss function (nn.CrossEntropyLoss)
See what's implemented at https://github.com/yzyouzhang/AIR-ASVspoof/blob/master/train.py

Correct me if I'm wrong, but the implementation is inaccurate.

If you need more explanation
Please get in touch with me at avishai11900@gmail.com.

Best regards,
Avishai
